ragengine/vector_store/faiss_store.py

```diff
@@ -0,0 +1,228 @@
+import os
+from typing import Dict, List
+
+import faiss
+from llama_index.core import Document as LlamaDocument
+from llama_index.core import (StorageContext, VectorStoreIndex, load_index_from_storage)
+from llama_index.core.storage.index_store import SimpleIndexStore
+from llama_index.core.storage.docstore.types import RefDocInfo
+from llama_index.vector_stores.faiss import FaissVectorStore
+
+from models import Document
```

Comment:
It looks like `models.py` was forgotten to be added to this pr.

Reply:
Added